next.config.js

```diff
@@ -1,6 +1,5 @@
-import "./env/server.ts"
-
+//@ts-check
```

Comment:
maybe, unsure if this works though
```suggestion
// @ts-check
```

Reply:
I guess that line is not needed at all for now